src/authentication/create-account-details/index.tsx

```diff
@@ -68,24 +68,26 @@ export class CreateAccountDetails extends React.Component<Properties, State> {
             <ImageUpload
               onChange={this.trackImage}
               icon={this.renderImageUploadIcon()}
-              uploadText='Select or drag and drop'
               isError={Boolean(this.props.errors.image)}
               errorMessage={this.props.errors.image}
             />
           </div>
           {this.imageError && <Alert variant='error'>{this.imageError}</Alert>}
           <Input
             {...cn('input')}
-            label='What is your name?'
-            helperText='This will be your name that is visible to others on Zero'
+            label='Display Name'
+            wrapperClassName='create-account-details__input-wrapper'
```

Comment:
not sure how best to write the classNames here when using `bemClassName`, so I have added the full className for now.

Reply:
Yeah, they're just tools that make the general 99% case shorter. If you wanted you could add:
```
const c = bem('create-account-details');
```
at the top of the file alongside the `cn` definition. Then this becomes:
`wrapperClassName={c('input-wrapper')}`

I.E., `bem` and `bemClassName` aren't mutually exclusive and if they're defined close to each other then the duplication is localized.